pandas/_libs/index.pyx

```diff
@@ -606,25 +606,107 @@ cdef class BaseMultiIndexCodesEngine:
                        in zip(self.levels, zip(*target))]
         return self._codes_to_ints(np.array(level_codes, dtype='uint64').T)
 
-    def get_indexer(self, object target, object method=None,
-                    object limit=None):
+    def get_indexer(self, object target) -> np.ndarray:
+        """
+        Gets an indexer, i.e. set of indexes into `self`'s values for the
+        values in `target`, where -1 represents a value in `target` not existing
+        in (the cross-product of) `self.levels`
```

Comment:
This sentence is awkward.  How about something like "Return an array giving the positions of each value of `target` in `self.values`"?

Reply:
updated here and in the other docstring.  Let me know if you'd like further revision